packages/expo-video/src/VideoView.types.ts

```diff
@@ -22,6 +22,14 @@ export interface VideoViewProps extends ViewProps {
    */
   nativeControls?: boolean;
 
+  /**
+   * Determines whether the video should play inline or not.
+   * This property is relevant on mobile web browsers.
```

Comment:
We could be more descriptive here, let's get inspired by MDN 😉
```suggestion
   * Determines whether the video is to be played "inline", that is, within the element's playback area. Note that the absence of this attribute does not imply that the video will always be played in fullscreen.
   * This property is relevant only on mobile web browsers.
```
If you found out any differences in behavior between different browsers (especially Safari), it might be good to note that here too.